### ISSUE TITLE
[Getting Started] Only show setup by default to "new" installations

### ISSUE THREAD

**Issue by digitarald** [microsoft/vscode#119415]
_Quick Setup_ now shows as first category until one of the tasks is done. This can be confusing to returning users, as they don't expect VS Code to launch into any kind of setup mode.

One possible solution is similar to `isNewAppInstall` [provides](https://github.com/microsoft/vscode/blob/0eff78fc42e1a3b9445496610ee2a5c8179e94d5/src/vs/workbench/api/common/extHost.api.impl.ts#L302), the heuristic for showing the setup category by default should only trigger during the first day of installation.

**Comment by digitarald** [comment]
@JacksonKearl as we want to experiment with the Getting Started page in release; this additional heuristic would it more seamless for returning users while getting new users into the setup flow. Should we slot it for April or do you think it can make it into March?

**Comment by JacksonKearl** [comment]
The idea is to use `isNewAppInstall` to control whether to open to the index page or the first category?

**Comment by digitarald** [comment]
@JacksonKearl yes. Not necessarily using the exact method but the heuristic of limiting it to users within their first day of usage.

**Comment by JacksonKearl** [comment]
Was supposed to be fixed via 5bf977258c548760845c41f4a208d15207883f6d, that seems to not have fully worked, going to look more into next iteration.

**Comment by JacksonKearl** [comment]
Fixed a bit ago.

**Comment by JacksonKearl** [comment]
To verify: 

1. launch insiders with empty data dir, it should go into the "setup" mode. Without doing anything to check boxes off, close the getting started tab, then close the editor. 
1. Launch again, using the same profile as before. It should still go into the setup mode. 
2. try doing something that would check off a box (changing theme, for instance). now close the getting started editor and the window.
3. Launch again in the same profile, it should now open the getting started page, but not scroll anyhwere.

**Comment by andreamah** [comment]
During the second step, it doesn't go into setup mode again for some reason?

Video:
https://user-images.githubusercontent.com/31675041/121070980-9a1ef100-c78c-11eb-82da-3dfdcd3a407e.mp4

**Comment by rzhao271** [comment]
@JacksonKearl I also notice that step 2 of that verification seems to fail. For me, it goes to the Getting Started page.
If I don't close the Getting Started tab before closing the editor, though, the steps seem to work.

**Comment by JacksonKearl** [comment]
Oh, how strange... you're both on windows, correct?

**Comment by JacksonKearl** [comment]
I'm still not able to reproduce on Windows. This isn't critical so I'll kick it back to look into this iteration.

**Comment by andreamah** [comment]
Yeah, that's pretty strange :/ If you need help with verification + reproducing the issue, let me know!
